polonius-engine/src/output/datafrog_opt.rs

```diff
@@ -62,28 +64,20 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
 
         // variables, indices for the computation rules, and temporaries for the multi-way joins
         let subset = iteration.variable::<(Region, Region, Point)>("subset");
-        let subset_1 = iteration.variable_indistinct("subset_1");
-        let subset_2 = iteration.variable_indistinct("subset_2");
         let subset_r1p = iteration.variable_indistinct("subset_r1p");
         let subset_p = iteration.variable_indistinct("subset_p");
```

Comment:
For posterity's sake: I think we can also get by with only one `subset` variable